src/System.Configuration.ConfigurationManager/tests/System/Configuration/TempConfigurationHost.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Configuration;
+using System.Configuration.Internal;
+using System.Reflection;
+
+public class TempConfigurationHost : DelegatingConfigHost {
```

Comment:
Nit: for all of these braces, our style in corefx is generally to put these on the next line.
https://github.com/dotnet/corefx/blob/bceb1d4d11b7f374498f15184499ff77f2c127ba/Documentation/coding-guidelines/coding-style.md#L10